sparse/usr/sbin/adbd-functionfs.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+mkdir -p /dev/usb-ffs
+chmod 0770 /dev/usb-ffs
+chown shell:shell /dev/usb-ffs
+mkdir -p /dev/usb-ffs/adb
+chmod 0770 /dev/usb-ffs/adb
+chown shell:shell /dev/usb-ffs/adb
+/bin/mount -t functionfs adb /dev/usb-ffs/adb -o uid=2000,gid=2000
```

Comment:
2000? Shouldn't this be some username and group name as if it is different on some android hardware this will blowup.